coffea/lookup_tools/jersf_lookup.py

```diff
@@ -98,9 +98,9 @@ def _evaluate(self, *args):
             eval_values.append(np.clip(eval_vals[eval_name], clamp_mins, clamp_maxs))
 
         # get parameter values
-        parm_values_central = self._parms[0][bin_tuple].flatten()
-        parm_values_up = self._parms[1][bin_tuple].flatten()
-        parm_values_down = self._parms[2][bin_tuple].flatten()
+        parm_values_central = np.array(self._parms[0][bin_tuple]).squeeze()
```

Comment:
Looks unrelated to ak1 or the mentioned bugfixes?

Reply:
It was getting an extra empty dimension for some reason.